package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "statvfs",
     "description": "node binding to POSIX statvfs (file system information)",
-    "version": "2.2.1",
+    "version": "2.2.2",
```

Comment:
NOTE: If we make this a v0.10 branch, we don't *need* to update this if the Node 0.10 users pull it from github directly?

Reply:
My concern is that if we lock down `nan` for ALL users via NPM publishing, we overly restrict their maximum version of `nan`.  Only our lower-micro-revs of node-0.10 get bitten by this. And we do have prior art here.  Or do I not understand the ~VERSION syntax correctly?

Reply:
Also NPM publishing latencies... are they a factor?

Reply:
Let me double check to confirm that v0.10 (and the npm being used there) behaves this way, but my understanding is that they won't be locked down to `~2.14.0`. If another dependency strictly needs another version such as `~2.15.0` npm will just install both versions of nan in their node_modules. I'll do a test and report back that this is indeed the case.

NPM publishing _should_ be instant.

Reply:
Confirmed that both modules will be installed in node v0.10 and npm 2.15.1. That being said I'm not opposed to a v0.10 branch either.